components/eam/src/physics/cam/modal_aer_opt.F90

```diff
@@ -788,6 +833,12 @@ subroutine modal_aero_sw(list_idx, dt, state, pbuf, nnite, idxnite, is_cmip6_vol
                   if (trim(spectype) == 'sulfate') then
                      do i = 1, ncol
                         burdenso4(i) = burdenso4(i) + specmmr(i,k)*mass(i,k)
+                        aburdenso4(i) = aburdenso4(i) + specmmr(i,k)*mass(i,k) + aspecmmr(i,k)*mass(i,k)
+                        if ((k .le. trop_level(i)) .and. (is_output_interactive_volc)) then ! in stratosphere
```

Comment:
Would it be better to remove "is_output_interactive_volc", so that in the simulation when is_output_interactive_volc = False we can also have the correct stratospheric/tropospheric sulfate burden?